RoadRangerMobileApp/RoadRangerMobileApp.Android/RoadRangerMobileApp.Android.csproj.user

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AndroidDesignerPreferredTheme>Theme.DeviceDefault.NoActionBar.Fullscreen</AndroidDesignerPreferredTheme>
     <SelectedDevice>Android_Accelerated_x86_Oreo</SelectedDevice>
-    <SelectedDevice>Android_Accelerated_x86_Oreo</SelectedDevice>
-    <DefaultDevice>Android_Accelerated_x86_Oreo</DefaultDevice>
+    <SelectedDevice>Android_Accelerated_x86_Nougat</SelectedDevice>
+    <DefaultDevice>Android_Accelerated_x86_Nougat</DefaultDevice>
```

Comment:
Why change default?

Reply:
This changes according to my emulator device.